lib/matplotlib/tests/test_category.py

```diff
@@ -187,6 +187,37 @@ def test_plot_1d_missing(self):
         self.axis_test(ax.yaxis, self.dmticks, self.dmlabels, self.dmunit_data)
 
     @cleanup
+    @pytest.mark.usefixtures("data")
+    @pytest.mark.parametrize("bins",
```

Comment:
I know I'm being a pain, but would you mind labeling the different parameterizations? It's the `id` kwarg on the decorator, see line 22 for an example.